frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,36 @@
-import { Button } from '~/libs/components/components.js';
+import { Form, Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/app-route.enum';
+import {
+  type UserSignInRequestDto,
+  userSignInValidationSchema,
+} from '~/packages/users/users.js';
+
+import { DEFAULT_SIGN_IN_PAYLOAD } from './libs/constants.js';
+import { signInFields } from './libs/fields.js';
+import styles from './styles.module.scss';
 
 type Properties = {
-  onSubmit: () => void;
+  onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => (
-  <>
-    <h1>Sign In</h1>
-    <form>
-      <Button label="Sign in" />
-    </form>
-  </>
+const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => (
+  <div className={styles['form-wrapper']}>
+    <h3 className={styles.title}>Sign in</h3>
```

Comment:
Let's stick to a consistent code style. `styles.formWrapper` because we already used camel case.